app/controllers/users_controller.rb

```diff
@@ -1,7 +1,72 @@
 class UsersController < ApplicationController
+  before_action :set_user, except: [:index, :new, :create]
+  before_action :prevent_rash_skin, except: [:index, :new, :create, :edit, :show]
 
-  def list
+  def index
     @users = User.all
   end
-  alias :index :list
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    respond_to do |format|
+      if @user.save
+        format.html {redirect_to users_path}
+      else
+        format.html {render :new}
+      end
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    respond_to do |format|
+      if @user.update(user_params)
+        format.html { redirect_to users_path, notice: 'User was successfully updated.' }
+      else
+        format.html { render :edit }
+      end
+    end
+  end
+
+  def trim
```

Comment:
Yes. These actions are the bundled in rails `resources` package. http://www.informit.com/articles/article.aspx?p=1671632&seqNum=5